tests/unit/Rules/SymbolicLinkTest.php

```diff
@@ -14,9 +14,13 @@
 namespace Respect\Validation\Rules;
 
 use PHPUnit\Framework\TestCase;
+use Respect\Validation\Validatable;
 
 $GLOBALS['is_link'] = null;
 
+// This override is only needed for the test, since 
+// symbolic links cannot be mocked properly in PHP.
+// See this issue for more info: https://github.com/mikey179/vfsStream/issues/89
 function is_link($link)
```

Comment:
As a said in a different comment, you could've removed this function workaround and use a real symbolic link instead.